Tests/TSCUtilityTests/ProgressAnimationTests.swift

```diff
@@ -74,6 +62,19 @@ final class ProgressAnimationTests: XCTestCase {
         XCTAssertEqual(outStream.bytes.validDescription, "")
     }
 
+  #if !os(Windows) // PseudoTerminal is not supported in Windows
+    func testPercentProgressAnimationTTY() throws {
+        let output = try readingTTY { tty in
+            let animation = PercentProgressAnimation(stream: tty.outStream, header: "TestHeader")
+            runProgressAnimation(animation)
+        }
+
+        let startCyan = "\u{1B}[36m"
+        let bold = "\u{1B}[1m"
+        let end = "\u{1B}[0m"
+        XCTAssertMatch(output.spm_chuzzle(), .prefix("\(startCyan)\(bold)TestHeader\(end)"))
+    }
+
     func testNinjaProgressAnimationTTY() throws {
```

Comment:
Id like to see this working eventually, not my highest priority, but if someone else would take that on it would be great!